### DISCUSSION TITLE
Docs qtile error

### DISCUSSION THREAD

**Discussion by cr1ogen** [qtile/qtile#4978] [General]
The configuration options are the same for the built-in CurrentLayoutIcon and WindowName widgets?
https://docs.qtile.org/en/latest/manual/ref/widgets.html#currentlayouticon
https://docs.qtile.org/en/latest/manual/ref/widgets.html#windowname

**Comment by elParaguayo** [comment]
They don't look the same to me. They will have a lot of the same options as they are based on the base `_TextBox` class but there are differences.

**Reply by cr1ogen** [reply]
Ok Thanks, I wasn't sure that's why I didn't want to make an issue
